comp/swerve100/src/main/java/org/team100/frc2024/RobotContainer.java

```diff
@@ -171,11 +171,8 @@ public RobotContainer(TimedRobot robot) throws IOException {
 
     
 
-        VisionDataProvider visionDataProvider = new VisionDataProvider(
-                m_layout,
-                poseEstimator,
-                poseEstimator::getEstimatedPosition);
-        visionDataProvider.enable();
+        NotePosition24ArrayListener notePositionDetector = new NotePosition24ArrayListener();
```

Comment:
you shouldn't remove the vision data provider.

Reply:
Also fixed